lib/components/RecentBlocks.js

```diff
@@ -8,11 +8,24 @@ import {
   ExpandedDataRow,
   Text
 } from '@bpanel/bpanel-ui';
-import { pick } from 'underscore';
+import { pick, omit } from 'underscore';
```

Comment:
Should we migrate to `lodash` and use it as a `peerDependency` to reduce bundle size?

Reply:
yeah, good call. Missed that this was still on underscore 👍